lib/note-editor/index.tsx

```diff
@@ -124,13 +124,13 @@ export class NoteEditor extends Component<Props> {
 
     const isTrashed = !!note.deleted;
 
-    const lastUpdatedDate = lastUpdated
-      ? new Date(lastUpdated).toLocaleString()
-      : 'Unknown';
-
     return (
       <div className="note-editor theme-color-bg theme-color-fg">
-        <div className="last-sync">Last synced: {lastUpdatedDate}</div>
+        <div className="last-sync">
```

Comment:
I left the div so we would keep some of the padding at the top of the note.

Reply:
Does the note text bounce when we go from unknown to known?

Also, I feel like we can eventually make sure the status is always known, so this should be moot. Leaving it in could be a decent reminder that our work is unfinished.

The effect I'm after with the bounce is the difference between not rendering this vs. adding `opacity: 0` or something like that so it takes the same vertical space but isn't visible.

I did not test to find out - trying to get through the PR queue…

Reply:
The "bounce" would be the decider for me. Ideally, we would hide the div entirely when it's unknown but more importantly, we don't want notes jumping around.